src/App.tsx

```diff
@@ -1,25 +1,101 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+
+  const changeUser = (userId: number) => setSelectedUserId(userId);
+
+  const getCurrentTodos = () => {
+    getTodos()
+      .then((currentTodosFromServer) => {
+        setTodos(currentTodosFromServer);
+      });
+  };
+
+  useEffect(() => {
+    getCurrentTodos();
+  }, []);
+
+  const selectHandler = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    if (event.target.value === 'all') {
+      getCurrentTodos();
```

Comment:
move this function TodoList component and use todos from props instead of getCurrentTodos() or getTodos() functions